quadproduction/OpenPype
# #366: bugfix: open saved as file when file was not saved yet

## Changelog Description
Add a try / except in photoshop script to return empty value instead of a string error when the current file has not been saved yet. The value is then intercepted by the `current_file` host function, which now return an empty value in this case instead of an empty string, which was previously considered as a valid file name and was causing an error relatif to the file extension at save.

## Testing notes:
1. Open Photoshop through OpenPype
2. Create a new scene without saving it
3. Create a new workfile with the "save as" button
4. After saving, the soft should automatically open the file correctly.
